exp_output.txt

```diff
@@ -1,13 +1,49 @@
-Prince Aidan of Khanduras(1) vs Fallen
-Prince Aidan of Khanduras(1) vs Fallen
-Prince Aidan of Khanduras(1) vs Zombie
-Prince Aidan of Khanduras(1) vs Fallen
-Prince Aidan of Khanduras(2) vs Fallen
-Prince Aidan of Khanduras(2) vs Zombie
-Prince Aidan of Khanduras(2) vs Fallen
-Prince Aidan of Khanduras(3) vs Blood Raven
-The hero won.
-Prince Aidan of Khanduras: LVL8
-   HP: 65/65
-  DMG: 10
-  ACD: 0.526127
+╔════════════════════════════╗
+║████████████████████████████║
+║░░░░░░░░░░░░░░░░░░░░░░░░░░░░║
+║░░┣┫██░░M░M░░░░░░░░░░░░░░░░░║
+║░░░░░░░░░░░░░░░░░░░░░░░░░░░░║
+║░░░░░░░░M░░░░░░░░░░░░░░░░░░░║
+║██████████████████░░░░░░░░░░║
+╚════════════════════════════╝
+Command: ╔════════════════════════════╗
+║████████████████████████████║
+║░░░░░░░░░░░░░░░░░░░░░░░░░░░░║
+║░░░░██░░M░M░░░░░░░░░░░░░░░░░║
+║░░┣┫░░░░░░░░░░░░░░░░░░░░░░░░║
+║░░░░░░░░M░░░░░░░░░░░░░░░░░░░║
+║██████████████████░░░░░░░░░░║
+╚════════════════════════════╝
```

Comment:
'Command:' után egy sortörés szerintem elférne. Vagy a kirajzolás előtt.
A kódban is meg nyilván az elvárt kimeneti file-ban is.